app/assets/javascripts/admin/addon/components/dashboard-new-features.gjs

```diff
@@ -5,17 +5,33 @@ import { htmlSafe } from "@ember/template";
 import { ajax } from "discourse/lib/ajax";
 import i18n from "discourse-common/helpers/i18n";
 import { bind } from "discourse-common/utils/decorators";
+import AdminConfigAreaCard from "admin/components/admin-config-area-card";
 import DashboardNewFeatureItem from "admin/components/dashboard-new-feature-item";
 
 export default class DashboardNewFeatures extends Component {
   @tracked newFeatures = null;
+  @tracked groupedNewFeatures = null;
   @tracked isLoaded = false;
 
   @bind
   loadNewFeatures() {
     ajax("/admin/dashboard/whats-new.json")
       .then((json) => {
-        this.newFeatures = json.new_features;
+        const items = json.new_features.reduce((acc, feature) => {
+          const key = feature.released_at
+            ? moment(feature.released_at).format("YYYY-MM")
+            : moment(feature.created_at).format("YYYY-MM");
+          acc[key] = acc[key] || [];
+          acc[key].push(feature);
+          return acc;
+        }, {});
+
+        this.groupedNewFeatures = Object.keys(items).map((date) => {
+          return {
+            date: moment.tz(date, "YYYY-MM").format("MMMM YYYY"),
```

Comment:
This doesn't seem right, the `moment.tz` second argument is the timezone, which should be `this.currentUser.timezone`